packages/block-editor/src/components/block-list/style.scss

```diff
@@ -340,6 +328,21 @@
 	}
 }
 
+// Spotlight mode.
+.is-focus-mode .block-editor-block-list__block {
+	opacity: 0.5;
+	transition: opacity 0.1s linear;
+	@include reduce-motion("transition");
+
+	&.is-selected,
+	&.is-selected .block-editor-block-list__block,
+	&.is-multi-selected,
+	&.is-multi-selected .block-editor-block-list__block,
+	&.has-child-selected {
```

Comment:
These selectors should also make it much easier to understand what's going on. :)